_layouts/default.html

```diff
@@ -56,6 +56,7 @@
                     <li><a href="{{ root_path }}#quick-start">Quick Start</a></li>
                     <li><a href="{{ root_path }}#download">Download</a></li>
                     <li><a href="//formats.kaitai.io/">Format Gallery</a></li>
+                    <li{% if page.url == '/news/' %} class="active"{% endif %}><a href="/news">News</a></li>
```

Comment:
And it's weird to link to https://kaitai.io/news without the trailing slash because it's permanently redirected to https://kaitai.io/news/ (with the trailing slash).